UnitTests/Tests/Formatters/TRTemperatureComparisonFormatterSpec.m

```diff
@@ -0,0 +1,34 @@
+#import "UnitTests-Swift.h"
+#import "TRTemperatureComparisonFormatter.h"
+
+@interface TRTemperatureComparisonFormatter (Tests)
+
++ (NSCalendar *)calendar;
+
+@end
+
+SpecBegin(TRTemperatureComparisonFormatter)
+
+NSDate* (^dateFromString) (NSString*) = ^NSDate* (NSString *dateString) {
```

Comment:
No problem on the pointers.

Because of the way I'm testing this right now I don't think we can. To keep a consistent time zone in test I've made a category up on LN4 to expose the private calendar, then on LN21 I set it's time zone to a specific one. There's likely other ways to accomplish this but for now this does the trick. We can probably set up the class a little better for testing when converting it to Swift.